Modules/_sqlite/statement.c

```diff
@@ -96,7 +96,7 @@ int pysqlite_statement_create(pysqlite_Statement* self, pysqlite_Connection* con
     Py_BEGIN_ALLOW_THREADS
     rc = sqlite3_prepare_v2(connection->db,
                             sql_cstr,
-                            -1,
+                            sql_cstr_len + 1,
```

Comment:
@erlend-aasland Can you check out the compiler warnings?

Reply:
Oops, thanks for the heads up!